stripe/api_requestor.py

```diff
@@ -12,6 +12,7 @@
 from stripe.multipart_data_generator import MultipartDataGenerator
 from stripe.six.moves.urllib.parse import urlencode, urlsplit, urlunsplit
 from stripe.stripe_response import StripeResponse
+from stripe.request_metrics import RequestMetrics
```

Comment:
Can you try to arrange this as alphabetically as possible (since it's already roughly in that order)? Just swap it with the line above.

Reply:
Done!